examples/event_handling/pong_sgskip.py

```diff
@@ -316,10 +316,7 @@ def on_redraw(event):
 def start_anim(event):
     canvas.mpl_disconnect(start_anim.cid)
 
-    def local_draw():
-        if animation.ax.get_renderer_cache():
-            animation.draw()
-    start_anim.timer.add_callback(local_draw)
+    start_anim.timer.add_callback(animation.draw)
```

Comment:
This is changing the semantics to call `draw()` unconditionally. I don't claim to understand why the original code did not draw if there is a renderer_cache, but we should now why it's been there and can now leave it out (@tacaswell you wrote this originally).

Semi-OT: This removes the only usage of `Axes.get_renderer_cache`, which is undocumented by the way. Do we still need to expose this? If yes, this needs documentation, if no we should deprecate it.

Reply:
I suspect that this was a hedge to make sure we were actually drawn once before running because the `draw` method on Animation requires use `Axes.draw_artist` which in turn requires the Axes to have been drawn once to get a cached render.

However, we have made a lot of progress in pushing the caching of renderers into exactly 1 place and only access them via a function that will fabricate one if needed so I suspect we may no longer need this.

Reply:
See above: https://github.com/matplotlib/matplotlib/pull/23494#discussion_r931949547

Reply:
See https://github.com/matplotlib/matplotlib/pull/23202 for removal of the `Axes.get_renderer_cache`